nmtwizard/preprocess/sampler.py

```diff
@@ -256,7 +256,7 @@ def _select_lines(f):
             # 1  if full sample (lines_kept == lines_count or no gsample)
             # >1 if oversampling (lines_kept > lines_count)
             # 0  if undersampling (lines_kept < lines_count)
-            min_occurrence = not gsample or int(f.lines_kept / f.lines_count)
+            min_occurrence = int(f.lines_kept / f.lines_count) or int(not gsample)
```

Comment:
Is it expected that the commit "Fix oversampling with gsample 0" is included in this branch? I suggest making another PR with this change for clarity.

Reply:
Yes, you are right, I'll do it in another PR